docs/process.md

```diff
@@ -2,39 +2,23 @@
 
 # Processes
 
-In Nextflow, a **process** is the basic processing primitive to execute a user script.
+In Nextflow, a **process** is a function that is specialized for executing scripts in a scalable and portable manner.
 
-The process definition starts with the keyword `process`, followed by process name and finally the process body delimited by curly braces. The process body must contain a string which represents the command or, more generally, a script that is executed by it. A basic process looks like the following example:
+Here is an example process definition:
```

Comment:
Because the process page is still a bit reference-ish, I feel like a small example is needed at the top to give the user a taste of what is coming. I would be more amenable to removing this example if this page were more focused as a guide

Right now, this page goes through all the variations of the script/shell/exec section, then the stub, before it finally gets to inputs/outputs, and then it simply goes through each input/output type exhaustively. This initial example is basically a band-aid until the overall page is improved

Reply:
Fair - looking at it on the deploy preview it's not as noticeable. I'll resolve this one.